src/pages/ProcessInstanceDetail/hooks/useInstanceData.ts

```
@@ -267,6 +282,12 @@ export const useInstanceData = (
   // before the first finishes, doubling the load and causing state flicker.
   const isFetchingRef = useRef(false);
 
+  // Latest caller-provided history callback. Stored in a ref so refreshes
+  // always read the freshest closure without forcing the whole hook to
+  // re-subscribe to `options`.
+  const onHistoryPartialRef = useRef(options?.onHistoryPartial);
+  onHistoryPartialRef.current = options?.onHistoryPartial;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/pages/ProcessInstanceDetail/hooks/useInstanceData.ts:289:3
  287 |   // re-subscribe to `options`.
  288 |   const onHistoryPartialRef = useRef(options?.onHistoryPartial);
> 289 |   onHistoryPartialRef.current = options?.onHistoryPartial;
      |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  290 |
  291 |   // ── Subprocess element statistics ─────────────────────────────────────────
  292 |   const fetchSubprocessStats = useCallback(

(react-hooks/refs)